src/character.h

```diff
@@ -651,6 +651,8 @@ class Character : public Creature, public visitable<Character>
         /** Returns a string of missed requirements (both stats and skills) */
         std::string enumerate_unmet_requirements( const item &it, const item &context = item() ) const;
 
+        void set_item_or_stack_favorite( const item *it, const bool favorite );
```

Comment:
I'd inline this function to avoid further bloating the `Character` class. It's used only once.

Reply:
I was likely to reuse that function elsewhere when expanding the feature to other menus (advanced inventory at least), but that's fair, I inlined it in my latest commit.